cowprotocol/solver_dashboard/solver_rewards_totals_1821285.sql

```diff
@@ -0,0 +1,107 @@
+-- This query gets all the rewards awarded to each solver.
```

Comment:
This seems to count rewards prior to CIP-20. If so, we should very explicitly highlight this, and imo, i don't actually think it is useful anymore and maybe should be deprecated.